deepomatic/api/http_helper.py

```diff
@@ -121,9 +121,11 @@ def __init__(self, app_id=None, api_key=None, verify_ssl=None,
 
         headers = {
             'User-Agent': self.user_agent,
-            'X-APP-ID': self.app_id,
             'X-API-KEY': self.api_key,
         }
+        if self.app_id:
+            headers['X-APP-ID'] = self.app_id
```

Comment:
You will need to pass your own headers into the constructor. Or create a function :
```
def default_headers(self):
    return {
            'User-Agent': self.user_agent,
            'X-APP-ID': self.app_id,
            'X-API-KEY': self.api_key,
        }
```

You can thus override it later.